core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -379,9 +476,9 @@ object ConsumerGroupCommand {
   }
 
   class ConsumerGroupCommandOptions(args: Array[String]) {
-    val ZkConnectDoc = "REQUIRED (only when using old consumer): The connection string for the zookeeper connection in the form host:port. " +
+    val ZkConnectDoc = "REQUIRED (unless new-consumer is used): The connection string for the zookeeper connection in the form host:port. " +
```

Comment:
I agree it's not super clear. Maybe we should just avoid saying it's required since it saves us from needing to qualify? You can try to fix this if you want, but I'd be ok just leaving it as it is since it seems orthogonal to the rest of this.